### PR TITLE
Request to merge the new class feature to master

### PR DESCRIPTION
**Pull request by BonkeGcobo**
We changed the Awesome books code, to be in classed instead of objects and arrays

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+     </div>`;
+      bookSec.insertAdjacentHTML('beforeend', eachBook);
+      document.querySelectorAll('.removebtn').forEach((btn) => {
+        btn.addEventListener('click', () => {
+          this.remove(btn.dataset.id);
+        });
+      });
+    });
+  }
+}
```
Comment (henatan99) [inline app.js:50]: - [x] I think this method is buggy. When I create some books it gets stored in the local storage and displayed in the DOM properly. But, when I refresh the page, the local storage data persists but the list of books disappears from the DOM. Kindly fix this issue! :pray: 

Screenshot from 2021-11-25 11-17-51

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @BonkeGcobo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (BonkeGcobo) [reply app.js:50]: Quick question, when I reload, the localStorage should be cleaned as well/ or the list of books shouldn't dissapear?

Comment (henatan99) [reply app.js:50]: Hi @BonkeG, thank you for asking. The list of books shouldn't disappear. The localstorage is completely ok.

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @BonkeG @ElijahTobs 

Great job making the changes suggested by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] In your README file I would suggest adding a screenshot of your app view and a brief description of the app to the portion shown in the image below. You can also remove it entirely from your README if you do not feel you are ready to add it. This would make your README more professional

Screenshot 2021-11-25 at 11 43 34

Cheers and Happy coding!👏👏👏

**Pull request merged**